articles/azure-monitor/app/javascript.md

```diff
@@ -143,6 +143,12 @@ By default, this SDK will **not** handle state-based route changing that occurs
 
 Currently, we offer a separate [React plugin](#react-extensions) which you can initialize with this SDK. It will also accomplish route change tracking for you, as well as collect [other React specific telemetry](https://github.com/microsoft/ApplicationInsights-JS/blob/17ef50442f73fd02a758fbd74134933d92607ecf/extensions/applicationinsights-react-js/README.md).
 
+> ***Note:*** Only use `enableAutoRouteTracking: true` if you are **not** using the React Plugin. Both are capable of sending new PageViews when the route changes. If both are enabled, duplicate PageViews may be sent.
+
+## Configuration: autoTrackPageVisitTime
+
+By setting `autoTrackPageVisitTime: true`, the time users spend on each page is tracked. On each new PageView, the duration the user spent on the *previous* page is sent as a [custom metric](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-custom-overview) named `PageVisitTime`. This custom metric is viewable in the [Metrics Explorer](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-getting-started) as a "log based metric".
```

Comment:
It just clicked with me -- it's sent as a metric! Does that mean it's not available in the underlying logs? How would a user do meaningful analysis (slide and dice) in the usage experience if it's a metric?

Reply:
No curated experiences are built around this particular metric and so everything would be very DIY